src/ngraph/op/lrn.cpp

```diff
@@ -24,7 +25,7 @@ using namespace ngraph;
 const string op::LRN::type_name{"LRN"};
 
 op::LRN::LRN(const Output<Node>& arg, double alpha, double beta, double bias, size_t size)
-    : LRN(arg, op::Constant::create(element::i32, Shape{1}, {1}), alpha, beta, bias, size)
+    : LRN(arg, op::Constant::create(element::u64, Shape{1}, {1}), alpha, beta, bias, size)
```

Comment:
Can we make this `i64` instead? That's what we've generally standardized on for shape-like tensors, including those that represent axis sets (even though the internal representation is signed... it's complicated :))

Reply:
Done ;)